packages/path_provider/path_provider_linux/example/pubspec.yaml

```diff
@@ -3,17 +3,17 @@ description: Demonstrates how to use the path_provider_linux plugin.
 publish_to: "none"
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
-  path_provider: ^1.6.10
+  path_provider: ^1.6.24
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^0.1.3
+  cupertino_icons: ^1.0.1+1
 
 dependency_overrides:
```

Comment:
The example can't be run because of version conflicts. If I run `flutter pub get` in `example` :
```
Because every version of path_provider_linux from path depends on path_provider_platform_interface ^2.0.0-nullsafety and path_provider >=1.6.4 depends on path_provider_platform_interface ^1.0.1, path_provider_linux from path is incompatible with path_provider >=1.6.4.

So, because pathproviderexample depends on both path_provider ^1.6.24 and path_provider_linux from path, version solving failed.
pub get failed (1; So, because pathproviderexample depends on both path_provider ^1.6.24 and path_provider_linux from path, version solving failed.)
```

Adding this would fix it : 
```yaml
path_provider_platform_interface: ^2.0.0-nullsafety
```

Reply:
There are multiple ways to fix it, forcing `path_provider_platform_interface`'s version can be once, unsure of which is the best.

Reply:
It still is resulting in conflicts.

Reply:
Did you add it as `dependency_overrides` ?

Reply:
Oh yes, it did worked. Thanks!